health-check/src/health_check/exporters/supportconfig_exporter.py

```diff
@@ -534,7 +533,7 @@ def _append_value_to_dict(
 
     def write_metrics(self):
         metrics = self.merge_metrics()
-        filename = Path("/opt/metrics/metrics.json")
+        filename = Path("/tmp/metrics/metrics.json")
```

Comment:
I wonder here if it wouldn't be better to own `/opt/metrics` instead of moving this to `/tmp`. Maybe if we move the config to `/var/lib`, we could move this to `/var/lib` too. Either case, this is inside of a container, so right now it shouldn't matter. In the future, if this is not containerized in a separate container, we should IMO just own the `/opt` or `/var/lib` path by the RPM.

Reply:
The situation here is that `/opt` is a non-writtable persistent volume bringing the `supportconfig_exporter.py` and other files to the container.

Here in this code, we the process running inside the container tries to write inside `/opt` then it fails.

That's why I'm moving them to `/tmp`.